src/sonic-config-engine/portconfig.py

```diff
@@ -1,29 +1,48 @@
 #!/usr/bin/env python
 import os
 import sys
+import json
+import ast
+import re
+from collections import OrderedDict
 
+# Global Variable
+LANES = 4
 
 def get_port_config_file_name(hwsku=None, platform=None):
+
+    # check 'platform.json' file presence
+    port_config_candidates_Json = []
+    port_config_candidates_Json.append('/usr/share/sonic/hwsku/platform.json')
+    if hwsku:
+        if platform:
+	        port_config_candidates_Json.append(os.path.join('/usr/share/sonic/device', platform, hwsku, 'platform.json'))
+    port_config_candidates_Json.append(os.path.join('/usr/share/sonic/platform', hwsku, 'platform.json'))
+    port_config_candidates_Json.append(os.path.join('/usr/share/sonic', hwsku, 'platform.json'))
+
+    # check 'portconfig.ini' file presence
```

Comment:
If platform.json presents, we should skip the port_config.ini file parsing.

Reply:
Yes, that's exactly the code is doing. If it gets _**'platform.json'**_ file, it will return the file location and parse only that file. Otherwise, it will parse **_'portconfig.ini'_** file.